scss/detail.scss

```diff
@@ -0,0 +1,27 @@
+* {
+  font-family: $font-family;
+}
+
+.title {
+  font-size: 32px;
+  font-weight: 700;
+  margin-bottom: 3.25rem;
```

Comment:
We are sticking to `px` for now. Maybe in a near future, we'll start using `rem` and `em` 😄